### DISCUSSION TITLE
Open Source or private repository

### DISCUSSION THREAD

**Discussion by kobebryant432** [CORDEX-be2/ValEnsPy#17] [Admin]
Should the package be open source?

**Comment by kobebryant432** [comment]
In my opinion, it should be open source in the long run. But for now, it will remain a private repository.

**Reply by Ivanderkelen** [reply]
I agree with this idea, but might need discussion in a CORDEXbe 2 project meeting. Let's keep it private for now.

**Reply by kobebryant432** [reply]
Agreed, it will change over time.

**Comment by kobebryant432** [comment]
Relevant question from #18:
what to do with CLIMATE_GRID dataset: put on RMI Hortense partition? Important not to share publicly?
